api/app/models/spree/api/webhooks/order.rb

```diff
@@ -0,0 +1,17 @@
+module Spree
+  module Api
+    module Webhooks
+      module Order
+        def after_cancel
+          super
+          Spree::Webhooks::Endpoints::QueueRequests.call(event: 'order.cancel', payload: {})
```

Comment:
@damianlegawiec, there's no serializer for Order, and the serializers for Payment and Shipment are in Spree::V2::Storefront, so the payloads in this case are being hardcoded as an empty hash. Should we use Spree::V2::Storefront? What would be the payload for Order?

Reply:
@sebastian-palma missing serialized are being added via https://github.com/spree/spree/pull/11376